sdk/monitor/azure-monitor-query/CHANGELOG.md

```diff
@@ -16,6 +18,8 @@
 - `metric_namespace_name` is renamed to `fully_qualified_namespace`
 - `is_dimension_required` is renamed to `dimension_required`
 - `time_grain` is renamed to `granularity`
+- `interval` is renamed to `granularity`
```

Comment:
I might lose the necessary context -- so `interval` and `time_grain` are the same thing now?

Reply:
yes - they are the same thing

Reply:
we just ended up renaming them differently in different places of swagger somehow